src/main/java/seedu/duke/CommandType.java

```diff
@@ -0,0 +1,24 @@
+package seedu.duke;
+
+/**
+ * Represents the different types of command that the user can input
+ */
+public enum CommandType {
```

Comment:
Very nitpicky comment, but aren't enum names usually in simpler form? e.g. Command?